mcp/worker.ts

```
@@ -0,0 +1,1481 @@
+                        },
+                    });
+
+                case 'tools/call':
+                    if (!honcho) {
+                        return new Response(JSON.stringify(createJsonRpcResponse(requestData.id ?? null, undefined, createJsonRpcError(-32602, 'Missing API key'))), {
+                            status: 401,
+                            headers: { 'Content-Type': 'application/json' },
+                        });
+                    }
+
+                    const toolName = requestData.params?.name;
```

Biome (2.1.2)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Safe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)